projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -154,14 +200,23 @@ export class IgxCell {
         return args;
     }
 
-    public createDoneEditEventArgs(value: any, event?: Event): IGridEditDoneEventArgs {
+    public createCellEditDoneEventArgs(value: any, event?: Event): IGridEditDoneEventArgs {
```

Comment:
`createCellEditDoneEventArgs` already belongs to the `IgxCell` so adding `Cell` in the name is a bit pointless, but I agree the switch to `EditDone` is fine. Regardless, all internal so this comment can be safely ignored too